temporalcli/commandsgen/commands.yml

```diff
@@ -3439,6 +3439,160 @@ commands:
           - accepted
         required: true
 
+  - name: temporal workflow start-update-with-start
+    summary: Send an Update and wait for it to be accepted or rejected (Experimental)
+    description: |
+      Send a message to a Workflow Execution to invoke an Update handler, and wait for
+      the update to be accepted or rejected. If the Workflow Execution is not running, 
+      then a new workflow execution is started and the update is sent.
+
+      Experimental.
+
+      ```
+      temporal workflow start-update-with-start \
+        --update-name YourUpdate \
+        --update-input '{"update-key": "update-value"}' \
+        --workflow-id YourWorkflowId \
+        --type YourWorkflowType \
+        --task-queue YourTaskQueue \
+        --input '{"wf-key": "wf-value"}'
+      ```
+    option-sets:
+      # workflow-id is "required" (runtime check)
+      - shared-workflow-start
```

Comment:
We want `id-conflict-policy` to be required for update-with-start -- all the SDKs do this. (I wonder how exciting of an implementation challenge that is in the CLI options.)

Reply:
Added a runtime check making `id-conflict-policy` required